packages/container/lib/container.js

```diff
@@ -127,9 +135,86 @@ Container.prototype = {
    */
   lookupFactory(fullName, options) {
     assert('fullName must be a proper full name', this.registry.validateFullName(fullName));
+
+    if (isFeatureEnabled('container-factoryFor')) {
+      deprecate('Using "_lookupFactory" is deprecated. Please use container.factoryFor instead.', false, { id: 'container-lookupFactory', until: '2.12.0', url: 'TODO' });
```

Comment:
Must be `until: '2.13'`. 2.12 is an LTS release and must contain this deprecation.